lib/builder/webpack/client.config.js

```diff
@@ -166,6 +166,17 @@ module.exports = function webpackClientConfig() {
       config.plugins.push(new webpack.optimize.ModuleConcatenationPlugin())
     }
 
+    // Chunks size limit
+    // https://webpack.js.org/plugins/aggressive-splitting-plugin/
+    if (this.options.build.maxChunkSize) {
+      config.plugins.push(
+        new webpack.optimize.AggressiveSplittingPlugin({
+          minSize: this.options.build.maxChunkSize,
```

Comment:
Is there any reason to set `minSize` exactly equal to `maxChunkSize`?

Reply:
Yes, the `minSize` have a misleading name...
It's a threshold point. Only chunks bigger than the specified minSize will be splitted.

So, setting the minSize to 300 kB, it avoid to join smaller chunks in one of 300kB. I don't want interfere your original Nuxt chunks splitting.

The default options of "Agressive Splitting" are :
```js
{
  minSize: 30000, //Byte, split point. Default: 30720
  maxSize: 50000, //Byte, maxsize of per file. Default: 51200
  chunkOverhead: 0, //Default: 0
  entryChunkMultiplicator: 1, //Default: 1
}
```